resources/stage_4_repair/MTRT - MS Telemetry Removal/MTRT.cmd

```diff
@@ -0,0 +1,316 @@
+CD /D "%DATA%"
+
+REM Start Window 10 Only
+IF /I NOT "%WIN_VER:~0,9%"=="Windows 1" (GOTO:SKIP_10_TWEAKS)
+REM Disabled for Tron; it seems regular Flash also stores files here
+REM CALL :LOGTXT "   Kill Adobe Flash that ships with Windows 10"
+REM RD /S /Q "%APPDATA%\Adobe" >NUL 2>&1
+REM RD /S /Q "%APPDATA%\Macromedia" >NUL 2>&1
+REM TAKEOWN /F "%WINDIR%\System32\Macromed" /A /R /D Y >NUL 2>&1
+REM ICACLS "%WINDIR%\System32\Macromed" /GRANT ADMINISTRATORS:F /T /Q >NUL 2>&1
+REM RD /S /Q "%WINDIR%\System32\Macromed" >NUL 2>&1
+REM TAKEOWN /F "%WINDIR%\SysWOW64\Macromed" /A /R /D Y >NUL 2>&1
+REM ICACLS "%WINDIR%\SysWOW64\Macromed" /GRANT ADMINISTRATORS:F /T /Q >NUL 2>&1
+REM RD /S /Q "%WINDIR%\SysWOW64\Macromed" >NUL 2>&1
+
+CALL :LOGTXT "   Kill OneDrive integration"
+CALL :LOGCMD TASKKILL /F /IM  Explorer.exe /T
+CALL :LOGCMD %SystemRoot%\System32\OneDriveSetup.exe /Uninstall
+CALL :LOGCMD %SystemRoot%\SysWOW64\OneDriveSetup.exe /Uninstall
+REM These keys are orphaned after the OneDrive uninstallation and can be safely removed
+CALL :LOGCMD REG DELETE "HKCR\CLSID\{018D5C66-4533-4307-9B53-224DE2ED1FE6}" /F
+CALL :LOGCMD REG DELETE "HKCR%Wow6432Node%\CLSID\{018D5C66-4533-4307-9B53-224DE2ED1FE6}" /F
+CD /D "%USERPROFILE%\AppData\Roaming\Microsoft\Windows\Start Menu\Programs"
+CALL :LOGCMD DEL "OneDrive.lnk" /F /S /Q
+FOR %%D IN ("%LOCALAPPDATA%\Microsoft\OneDrive","%USERPROFILE%\OneDrive","%PROGRAMDATA%\Microsoft OneDrive","%SYSTEMDRIVE%\OneDriveTemp") DO (
+	CALL :LOGCMD TAKEOWN /F %%D /A /R /D Y
+	CALL :LOGCMD ICACLS %%D /GRANT ADMINISTRATORS:F /T /Q
+	CALL :LOGCMD DEL %%D /F /S /Q
+	CALL :LOGCMD RMDIR %%D /S /Q
+)
+CD /D "%WINDIR%\WinSxS"
+FOR /D %%O IN (*onedrive*) DO (
+	CALL :LOGCMD TAKEOWN /F %%O /A /R /D Y
+	CALL :LOGCMD ICACLS %%O /GRANT ADMINISTRATORS:F /T /Q
+	CALL :LOGCMD DEL %%O /F /S /Q
+	CALL :LOGCMD RMDIR %%O /S /Q
+)
+CALL :LOGCMD START Explorer.exe
+CD /D "%DATA%"
+
+CALL :LOGTXT "   Disable geolocation Service"
+CALL :LOGCMD SC STOP lfsvc 
+CALL :LOGCMD SC CONFIG lfsvc START= disabled
+
+CALL :LOGTXT "   Delete WAP Push Message Routing Service"
+CALL :LOGCMD SC STOP dmwappushservice
+CALL :LOGCMD SC DELETE dmwappushservice
+
+CALL :LOGTXT "   Delete Xbox Live Services"
+CALL :LOGCMD SC STOP XblAuthManager
+CALL :LOGCMD SC STOP XblGameSave
+CALL :LOGCMD SC STOP XboxNetApiSvc
+CALL :LOGCMD SC DELETE XblAuthManager
+CALL :LOGCMD SC DELETE XblGameSave
+CALL :LOGCMD SC DELETE XboxNetApiSvc
+
+CALL :LOGCMD REG ADD "HKLM\SOFTWARE\Microsoft\WcmSvc\wifinetworkmanager\features\%SID%" /T REG_DWORD /V FeatureStates /D 0x33c /F
+
+:SKIP_10_TWEAKS
+
+
+
+CALL :LOGTXT "   Adding registry tweaks"
+CD /D "%DATA%"
+FOR /F "eol=# tokens=%TOKENS% delims=	" %%A IN (Reg.ini) DO (
+	IF /I %%A==Y (
+		CALL :LOGCMD REG ADD "%%B" /T %%E /V %%C /D %%D /F
+	)
+)
+
+CALL :LOGTXT "   Blocking PersistentRoutes"
```

Comment:
I thought that was only true on windows 10, not 7 and 8? I can skip this step for windows 10.